.github/workflows/test_pip_install.yaml

```
@@ -22,21 +22,22 @@ on:
       
 jobs:
   test_release:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v2
-      - name: Get the tags
-        run: git fetch --all --tags
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          fetch-tags: true
       - name: Install Python's "build" package
-        run: python -m pip install build twine --user
+        run: python -m pip install build twine
       - name: Build binary wheel and source tarball
         run: python -m build --sdist --wheel --outdir dist/
       - name: Test Long Description
         run: twine check dist/*
```

For consistency with every other call using `python -m`:
```suggestion
        run: python -m twine check dist/*
```